cukinia

```diff
@@ -194,6 +194,18 @@ cukinia_log()
 	esac
 }
 
+ver2int() {                                                                     
```

Comment:
please read the original source before pushing changes.
normalize naming, style, comments, alignment with cukinia utility functions.
The function returns different versions for 1.0 and 1.0.0, it should return the same